src/sync.rs

```diff
@@ -0,0 +1,159 @@
+// Basic syncing for ActivityWatch
```

Comment:
Use triple /// instead to make actual rustdoc comments instead of just code comments